src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -173,7 +175,12 @@ public class DownloadPipelineArtifactTaskV2_0_0 : PipelineArtifactTaskPluginBase
                 {
                     throw new ArgumentNullException("Project Name cannot be null.");
                 }
-                Guid projectId = Guid.Parse(projectName);
```

Comment:
Was this variable (projectId) not used before?

Reply:
It is.  I recreated one that will be assigned by the TryParse() function instead.